package.json

```diff
@@ -17,11 +18,18 @@
   "bin": {
     "cucumber": "./node_modules/cucumber/bin/cucumber.js"
   },
+  "browserify": {
+    "transform": [
+      "babelify",
+      "brfs"
+    ]
+  },
   "scripts": {
     "lint": "eslint -c ./.eslintrc features/step_definitions/ features/support/",
     "test": "npm run lint && ./node_modules/cucumber/bin/cucumber.js features/ -p verify",
     "clean-test": "rm -rf test/cache",
-    "cucumber": "./node_modules/cucumber/bin/cucumber.js"
+    "cucumber": "./node_modules/cucumber/bin/cucumber.js",
+    "build-api-docs": "rm -rf build/docs && mkdir -p build/docs/tmp/src && cp -r node_modules/docbox/src/* build/docs/tmp/src && cp -r node_modules/docbox/css build/docs/ && cp -r docs/images build/docs && cp node_modules/docbox/index.html build/docs/tmp && cp docs/src/* build/docs/tmp/src/custom && mkdir -p build/docs/tmp/content && cp docs/*.md build/docs/tmp/content && cd build/docs/tmp && NODE_ENV=production browserify src/index.js | uglifyjs -c -m > ../bundle.js && babel src --out-dir lib && node lib/render.js ../index.html && cd .. && rm -rf tmp"
```

Comment:
hm can we move this to a script in `./script/build_docs.sh` or something? Kind of hard to read.